lib/commands/run.ts

```diff
@@ -116,8 +116,10 @@ export class RunCommandBase implements ICommand {
 			watchAllFiles: this.$options.syncAllFiles,
 			clean: this.$options.clean
 		};
-
-		await this.$liveSyncService.liveSync(deviceDescriptors, liveSyncInfo);
+		const projectFilesConfig: IProjectFilesConfig = {
+			configuration: this.$options.release ? Configurations.Release.toLowerCase() : Configurations.Debug.toLowerCase()
+		};
+		await this.$liveSyncService.liveSync(deviceDescriptors, liveSyncInfo, projectFilesConfig);
```

Comment:
Do you need to pass `projectFilesConfig` to `liveSync` method - `liveSync` is called only for debug configuration (you can check some lines above that in case `--release` is passed, we'll never get here. Isn't the default behavior of file mappings (using the debug configuration) working fine in this case?